### PR TITLE
Simple make CI

### PR DESCRIPTION
**Pull request by aminvakil**
Fixes https://github.com/micheleg/dash-to-dock/issues/1654

### CODE REVIEW

**Comment by aminvakil** [comment]
Currently run actions can be viewed from https://github.com/aminvakil/dash-to-dock/actions.

Merging #1648 fixes the currently broken make obviously and can be verified [here](https://github.com/aminvakil/dash-to-dock/actions/runs/1738576432) too.

File: .github/workflows/make.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+name: Make
+'on':
+  pull_request:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '0 0 * * *'
```
Comment (3v1n0) [inline .github/workflows/make.yml:9]: There's no need for having a cron check here... I think it's fine on pushes and cron.

File: .github/workflows/make.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+name: Make
+'on':
+  pull_request:
+  push:
+    branches:
+      - master
```
Comment (3v1n0) [inline .github/workflows/make.yml:7]:
```suggestion
      - master
      - ubuntu-dock
```

**Review by 3v1n0 — CHANGES_REQUESTED** [review]

File: .github/workflows/make.yml (hunk 1 above)
Comment (aminvakil) [reply .github/workflows/make.yml]: Fixed in https://github.com/micheleg/dash-to-dock/pull/1657/commits/f219115cde96a1743d73d3aa8da7378afe7f2216

**Comment by 3v1n0** [comment]
Thanks, i've also added some eslint checks

**Pull request merged**